common/djangoapps/third_party_auth/urls.py

```diff
@@ -2,12 +2,14 @@
 
 from django.conf.urls import include, url
 
-from .views import inactive_user_view, lti_login_and_complete_view, post_to_custom_auth_form, saml_metadata_view
+from .views import (inactive_user_view, lti_login_and_complete_view,
+                    post_to_custom_auth_form, saml_metadata_view, IdPRedirectView)
 
 urlpatterns = [
     url(r'^auth/inactive', inactive_user_view, name="third_party_inactive_redirect"),
     url(r'^auth/custom_auth_entry', post_to_custom_auth_form, name='tpa_post_to_custom_auth_form'),
     url(r'^auth/saml/metadata.xml', saml_metadata_view),
     url(r'^auth/login/(?P<backend>lti)/$', lti_login_and_complete_view),
+    url(r'^auth/idp_redirect/(?P<provider_slug>.*)', IdPRedirectView.as_view(), name="idp_redirect"),
```

Comment:
This matches every string, including /. In practice, the slug is probably only going to be within `[\w-]+`

Reply:
Ah, good point. I agree that that regex is more sensible.